src/Filters/RoleFilter.php

```
@@ -56,7 +59,7 @@ public function before(RequestInterface $request, $params = null)
 				return redirect()->to($redirectURL)->with('error', lang('Auth.notEnoughPrivilege'));
         	}
         	else {
-        		throw new \RuntimeException(lang('Auth.notEnoughPrivilege'));
+        		return redirect()->to('login')->with('error', lang('Auth.notEnoughPrivilege'));
```

Probably would be better if this checked `$config->silent` to determine whether it should throw an exception or return an error message.

---

Don't understand it. silent defaults to false.
It's then checked and, if it's true, it returns an error. But if it's false (don't be silent as I understand) it throws.
By silent you mean "let the user know/not know" or is it meant to be a developer aid so it throws when error appear?